tests/lac/vector_type_traits_is_serial_08.output

```diff
@@ -0,0 +1,3 @@
+
+DEAL::1
```

Comment:
Since this test is for `PETSc`, the output file name must be adjusted to `vector_type_traits_is_serial_06.with_petsc=true.output`